openshift/patches/0003-quarkus-productized.patch

```diff
@@ -58,7 +58,7 @@ index 2bc2df55..e18ac0f3 100644
 -    <quarkus.platform.group-id>io.quarkus.platform</quarkus.platform.group-id>
 -    <quarkus.platform.version>2.16.0.Final</quarkus.platform.version>
 +    <quarkus.platform.group-id>com.redhat.quarkus.platform</quarkus.platform.group-id>
-+    <quarkus.platform.version>2.13.5.Final-redhat-00003</quarkus.platform.version>
++    <quarkus.platform.version>2.13.5.Final-redhat-00002</quarkus.platform.version>
```

Comment:
This change should be done twice -- for both CE and HTTP template.

Reply:
nice catch..